content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+
+[Back to the index](#index)
+
+### Datasets 
+
+So the important question is what should the dataset look like? This will quite depend upon on your [Research or business questions](#Research-or-business-questions) to hand.
+
+Having a look at how others have created their datasets for the various domains or topics or research subjects, can also help in the process. See this [Datasets resource](https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/data/datasets.md) which is a mix of both clean and unclean datasets. This can also answer the more specific question, what file format should the dataset be saved in? CSV? JSON? or any other know file formats.
+
+They can throw light on types of data (features) to collect or not to collect, and in what form should they be in when collected such that it's useful for the next steps in the process.
+
+[Back to the index](#index)
+
+### Data Generation
+
+Data generation is also a good MVP step/process, and also could be incorporated as one of your steps during the first few iterations - if you discover or know from observation that the data to hand is not sufficient or correct or even not usable for the end-purpose (see [Research or business questions](#Research-or-business-questions)).
+
+It is often considered when a domain or field has limited data or previous experiments or exercises have resulted in a small dataset but the dataset may be some what representative of the domain or problem that is being solved - then Data Generation using existing could be take resort to. This of course has it's pros and cons - you soon get a lot of data but then is the data biased and in what way? Is it balanced or not, if not what to do - all these kinds of questions come to the surface.
+
+This topic may need a section on it's own but there are many resources and facilities propping up these days that do exactly this, see this resource on [Data Generation](https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/data/data-generation.md#data-generation).
+
+[Back to the index](#index)
+
+### Visualisation
+
+"A picture speaks a thousand words" or "A picture is worth a thousand words" or other variants - they all mean the same thing!
+
+There is a misconception that Data Visualisation takes place towards the end or during some other process in the end-to-end process. But the fact is, whenever you have data, no matter what state it is in, we can partially (if not fully) visualise it. And gain an insight into what it is representing and what state it is in. It is an art and not just a science to be able to do this, the science part of drawing pictures from it is more or less solved, but evaluating these pictures and diagrams is another skill. And depending on how well one knows their domain may or may not be able to make sense from them -- which again goes back to the [Research or business questions](#Research-or-business-questions).
```

Comment:
very good points. We should link here the "data viz guide" once it will be complete 👯

Reply:
I think I can link the Data Viz guide for now and then when we refactor we can come back and fix the link! Good point!

Reply:
Btw I changed all "visualisation" to "visualization" since it's found all across the various guides!